### ISSUE TITLE
Secret mask confuses subsequent output

### ISSUE THREAD

**Issue by kgibm** [openshift/installer#4523]
When the ~2KB secret is pasted in the macOS Terminal, the `*` characters from [AlecAivazis/survey](https://github.com/AlecAivazis/survey) that the openshift installer uses confuse subsequent output (see below). [This issue](https://github.com/AlecAivazis/survey/issues/306) has now been resolved upstream so please ensure you bump your version of `survey` at the next opportunity.

```
? Pull Secret [? for help] ***************************************************************************************************************************************************************************************************************************************************INFO Creating
infrastructure resources...         ****************************************************************************************************************************************************************************************************************************INFO Waiting 
up to 20m0s for the Kubernetes API at https://api.cluster-f1bd.sandbox1365.opentlc.com:6443... ***************************************************************************************************************************************************INFO API 
v1.17.1 up                               ***********************************************************************************************************************************************************************************************************************INFO Waiting 
up to 40m0s for bootstrapping to complete... ***************************************************************************************************************************************************************************************************************INFO Destroying 
the bootstrap resources...        ***********************************************************************************************************************************************************************************************************************INFO Waiting 
up to 30m0s for the cluster at https://api.cluster-f1bd.sandbox1365.opentlc.com:6443 to initialize... ******************************************************************************************************************************************************INFO Waiting up 
to 10m0s for the openshift-console route to be created... ***********************************************************************************************************************************************************************************************INFO Install 
complete!                            ***********************************************************************************************************************************************************************************************************************INFO To 
access the cluster as the system:admin user when using 'oc', run 'export KUBECONFIG=/home/[...]/cluster-f1bd/auth/kubeconfig' *************************************************************************************************************INFO Access 
the OpenShift web-console here: https://console-openshift-console.apps.cluster-f1bd.sandbox1365.opentlc.com ***************
```

**Comment by tsze-redhat** [comment]
Created Bugzilla
https://bugzilla.redhat.com/show_bug.cgi?id=1947067

**Comment by pruan-rht** [comment]
@kgibm I've tried it on iTerm2 and the output is much cleaner.  
### iTerm2
```
$ openshift-install create install-config  --dir=pruan0407b
? SSH Public Key /Users/pruan/.ssh/openshift-qe.pub
? Platform azure
? azure subscription id xxx
? azure tenant id xxx
? azure service principal client id xxx
? azure service principal client secret [? for help] **********************************
INFO Saving user credentials to "/Users/pruan/.azure/osServicePrincipal.json"
INFO Credentials loaded from file "/Users/pruan/.azure/osServicePrincipal.json"
? Region eastus
? Base Domain qe.azure.devcluster.openshift.com
? Cluster Name pruan0407a
? Pull Secret [? for help] *******************************************************************************INFO Install-Config created in: pruan0407b        ********************************************************
```
```
### default iTerm from MacOS
$ openshift-install create install-config  --dir=pruan0407iterm
? SSH Public Key /Users/pruan/.ssh/openshift-qe.pub
? Platform azure
INFO Credentials loaded from file "/Users/pruan/.azure/osServicePrincipal.json" 
? Region eastus
? Base Domain qe.azure.devcluster.openshift.com
? Cluster Name pruan0407iterm
? Pull Secret [? for help] *****************************************************************************************************************************************************INFO Install-Config created in: pruan0407iterm    ******************************************************************************************************************************prprprprpruaprprprpruaprprpruaprpruan@pruan@prprpruaprprprprprprprprprprprprprprpruaprpruaprpruapruaprpruan@Mapruan@prpruaprprprprprprprprprprpruaprprprprprprprprprprprprprprprprprpruan@prpruaprpruapruapruapruapruapruan@MacBpruan@MacBpruan@MacBpruan@pruan@pruapruapruapruapruaprpruapruan@prprpruan@MacBook-Pro ~/my_clusters $ 
```

**Comment by kgibm** [comment]
@pruan-rht Hi Peter, thanks. This isn't a persistent issue for me so I don't need a workaround. It was just a product improvement request.

**Comment by mtnbikenc** [comment]
Should be fixed in #4895
